src/components/signon/CreatePassword.tsx

```diff
@@ -0,0 +1,289 @@
+  const [passBorderColor2, setPassBorderColor2] = useState('#F7F7F9');
+
+  const [isWorking, setisWorking] = useState(false);
+  const [password, setPassword] = useState('');
+  const [password2, setPassword2] = useState('');
+
+  const Checkbox = (label: string) => {
+    return (
+      <View style={styles.checkboxContainer}>
+        <View style={styles.unchecked}></View>
+        <Text style={[styles.uncheckedDescription, {alignSelf: 'center'}]}>{label}</Text>
+      </View>
+    )
+  }
+
+  return (
+    // SignIn - SignUp
+    <SafeAreaView>
+      <TouchableWithoutFeedback onPress={Keyboard.dismiss} accessible={false}>
+        <View style={styles.container}>
+          <HeaderLogo style={styles.audiusLogoHeader} fill='#C2C0CC' />
+          <View style={styles.containerForm}>
+            <FormTitle></FormTitle>
+            <TextInput
+              style={[styles.input, { borderColor: passBorderColor }]}
+              placeholderTextColor='#C2C0CC'
+              underlineColorAndroid='transparent'
+              placeholder="Password"
+              autoCompleteType="off"
+              autoCorrect={false}
+              autoCapitalize='none'
+              //clearTextOnFocus={true}
+              enablesReturnKeyAutomatically={true}
+              maxLength={100}
+              textContentType="password"
+              secureTextEntry={true}
+              onChangeText={(newText) => {
+                // setPassword(newText)
+              }}
+              onFocus={() => { setPassBorderColor('#7E1BCC') }}
+              onBlur={() => { setPassBorderColor('#F7F7F9') }}
+            />
+            <TextInput
+              style={[styles.input, { borderColor: passBorderColor2, marginBottom: 10 }]}
+              placeholderTextColor='#C2C0CC'
+              underlineColorAndroid='transparent'
+              placeholder="Confirm Password"
+              autoCompleteType="off"
+              autoCorrect={false}
+              autoCapitalize='none'
+              //clearTextOnFocus={true}
+              enablesReturnKeyAutomatically={true}
+              maxLength={100}
+              textContentType="password"
+              secureTextEntry={true}
+              onChangeText={(newText) => {
+                // setPassword(newText)
+              }}
+              onFocus={() => { setPassBorderColor2('#7E1BCC') }}
+              onBlur={() => { setPassBorderColor2('#F7F7F9') }}
+            />
+            {Checkbox(messages.checks[0])}
+            {Checkbox(messages.checks[1])}
+            {Checkbox(messages.checks[2])}
+            {Checkbox(messages.checks[3])}
+            <Text style={styles.terms}>{messages.termsAndPrivacy}
+            <Text style={{color:'#CC0FE0'}}> {messages.terms}</Text>
+            <Text> {messages.and}</Text>
+            <Text style={{color:'#CC0FE0'}}> {messages.privacy}</Text>
+            </Text>
+            <TouchableOpacity
+            style={styles.formBtn}
```

Comment:
I think this needs to be run through Prettier, if you have it installed you can enable "Format on save" in VSCode or another editor